tests/micro/zephyr/test_zephyr.py

```diff
@@ -89,7 +89,7 @@ def _make_add_sess(temp_dir, model, zephyr_board, west_cmd, build_config, dtype=
 # The same test code can be executed on both the QEMU simulation and on real hardware.
 @tvm.testing.requires_micro
 @pytest.mark.skip_boards(["mps2_an521"])
-def test_add_uint(temp_dir, board, west_cmd, tvm_debug):
+def test_add_uint(workspace_dir, board, west_cmd, tvm_debug):
```

Comment:
For this and all the other tests - why do we have to use `workspace_dir`? If it's going to be reused every time, it would probably be cleaner to use the Python builtin fixutre `temp_dir`?

Reply:
`workspace_dir` is a custom path directory where it is in the tvm workspace, so user can easily access them even when they are testing in docker. Also, we have `tvm-debug` which we can use to keep the project for debugging purposes. I think it's a useful tool for developers.